.env.example

```diff
@@ -0,0 +1,9 @@
+DEPARTMENTS_TO_CHECK=75,92,93
```

Comment:
Let's add a `copy .env.example .env && vim .env` to the readme

Reply:
Done